packages/samples/test/output/grpc-kiosk-example/@typespec/openapi3/openapi.yaml

```diff
@@ -50,7 +50,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/RpcStatus'
-  /v1/kiosks/{id}:
+  '/v1/kiosks/{id}':
```

Comment:
Is this quoting actually required for correct parsing?

Reply:
It isn't but I think this library is being a little more cautious with what can be yaml keys. Someone just filed an issue exactly for that however [#493](https://github.com/eemeli/yaml/issues/493)

It would be nicer to not have those quotes but I don't think it is also that big of a deal.